lnrpc/devrpc/dev_server.go

```diff
@@ -342,3 +346,25 @@ func (s *Server) ImportGraph(ctx context.Context,
 
 	return &ImportGraphResponse{}, nil
 }
+
+// Quiesce initiates the quiescence process for the channel with the given
+// channel ID. This method will block until the channel is fully quiesced.
+func (s *Server) Quiesce(_ context.Context, in *QuiescenceRequest) (
+	*QuiescenceResponse, error) {
+
+	txid, err := lnrpc.GetChanPointFundingTxid(in.ChanId)
+	if err != nil {
+		return nil, err
+	}
+
+	op := wire.NewOutPoint(txid, in.ChanId.OutputIndex)
+	cid := lnwire.NewChanIDFromOutPoint(*op)
+	_, err = s.cfg.Switch.GetLink(cid)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO(proofofkeags): Add Link operation for initiating quiescence and
+	// implement the rest of this in those terms
+	return nil, fmt.Errorf("TODO(proofofkeags): Implement")
```

Comment:
Reorder commits so that we don't have to add + remove todos 🙏

Reply:
This was done to make the review easier to tell the story of why each thing was necessary. I find that designs are better when we work backwards from the demands rather than forwards from the tools.